RFC.md

```diff
@@ -153,10 +169,19 @@ The Tool Response Schema defines the structure of the data returned after a tool
 
 - **Execution Metadata:**
 
-  - **`execution_id`**: The globally unique execution identifier.
-  - **`duration`**: Execution time in milliseconds.
-  - **`finished_at`**: Timestamp marking the completion of the execution.
-  - **`success`**: Boolean flag indicating the success or failure of the execution.
+  - **`success`**: **REQUIRED** Boolean flag indicating the success or failure of the execution.
+  - **`execution_id`**: **REQUIRED** The globally unique execution identifier.
+[//]: # ( We could consider dropping both of these fields. The client can track duration and finished_at themselves. Those measurements incorporate round-trip time and any additional latency and in practice, that's what the client will care about.)
+  - **`duration`**: **Optional** Execution time in milliseconds.
+  - **`finished_at`**: **Optional** Timestamp marking the completion of the execution. In ISO-8601 format, must be UTC.
```

Comment:
Removed `finished_at` and marked `duration` as optional (for those servers that want to return detailed timing info)